<ui version="4.0" >
 <class>QmitkSimpleMeasurementControls</class>
 <widget class="QWidget" name="QmitkSimpleMeasurementControls" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>SimpleMeasurement</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel1_3_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="text" >
      <string>&lt;p align="center">Select one option&lt;/p></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbDistance" >
     <property name="toolTip" >
      <string>calculates the distance between two points</string>
     </property>
     <property name="text" >
      <string>Distance</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbAngle" >
     <property name="toolTip" >
      <string>calculates the angle between two lines</string>
     </property>
     <property name="text" >
      <string>Angle</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbPath" >
     <property name="cursor" >
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>calculates the distance between two or more points</string>
     </property>
     <property name="text" >
      <string>Path</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <property name="autoRepeat" >
      <bool>false</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="default" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>65</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="text" >
      <string>&lt;p align="center">Hold &lt;i>shift + click&lt;/i>&lt;br>to insert a point&lt;/p></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>390</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
